filer/static/filer/js/addons/table-dropzone.js

```diff
@@ -32,7 +33,9 @@
         var baseUrl;
         var baseFolderTitle;
         var updateUploadNumber = function () {
+            uploadText.addClass('hidden');
             uploadNumber.text(maxSubmitNum - submitNum + '/' + maxSubmitNum);
+            uploadText.removeClass('hidden');
```

Comment:
my guess is that text update is slow, so we hide it until it's updated. But `.text()` shouldn't be that slow actually?